openpdf/src/test/java/com/lowagie/text/pdf/PdfDocumentTest.java

```diff
@@ -0,0 +1,43 @@
+package com.lowagie.text.pdf;
+
+import java.util.LinkedList;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+import com.lowagie.text.Element;
+import com.lowagie.text.Paragraph;
+
+class PdfDocumentTest {
+
+    @Test
+    void testCreateWithAllElementsInOneCell() {
+        final PdfPTable table = new PdfPTable(2);
+        table.addCell("Lorem");
+        table.addCell("ipsum");
+        table.addCell("dolor");
+        table.addCell("sit");
+
+        final Paragraph mainParagraph = new Paragraph();
+        final Paragraph paragraph1 = new Paragraph("Text above table");
+        final Paragraph paragraph2 = new Paragraph("Text below table");
+
+        mainParagraph.add(paragraph1);
+        mainParagraph.add(table);
+        mainParagraph.add(paragraph2);
+
+        PdfPTable result = PdfDocument.createWithAllElementsInOneCell(mainParagraph);
+        assertThat(result.getRows().size(), equalTo(1));
```

Comment:
Might it would be better to add each `assert*` to a separate test (for example, using `@TestFactory`) so each failing check will not suppress successive ones?